docs/03. Role providers.md

```diff
@@ -93,7 +93,7 @@ If you are more confident with flat RBAC, the previous config can be re-written
 return [
     'zfc_rbac' => [
         'role_provider' => [
-            'ZfcRbac\Role\InMemoryRoleProvider' => [
+            \ZfcRbac\Role\InMemoryRoleProvider::class => [
```

Comment:
Thanks for updating the doc, however let's keep that for the end, as I suspect we'll need to do some major architectural changes :).